### ISSUE TITLE
[Issue tracker] make Batch mode visible only with `developer` permission 

### ISSUE THREAD

**Issue by christinerogers** [aces/Loris#9422]
Top-priority followup from #9339,  spun out of #9418 

Batch mode be visible to all who can see the module, as it is currently.  
It's designed for the dev/DCC team to handle and update issues -- End Users of Loris shouldn't be able to see / use Batch mode.  

what to do: 
- [ ] Hide the Batch Mode **button** unless the user has `issue_tracker_developer` permission

Consider doing #9423 at the same time - correct the button style 

If time / nice to have:  make sure the batch page doesn't load if the user doesn't have `issue_tracker_developer`

This solves the requirement for a permission restricting access, clearly defined in the original issue, and it was simply not done yet. 
I don't want this to block #9339 but it is still important and should be the next PR after #9339.  

@ay-bh please follow up with @racostas to ensure we are addressing this as a priority next PR, thank you very much

**Comment by regisoc** [comment]
Thanks for this @christinerogers ! As it is a new feature that we will experiment on, maybe starting with a new permission for beta testing might be more suited. What do you think @racostas @christinerogers @ay-bh ?

**Comment by christinerogers** [comment]
I think the minimum necessary here for Nov 1 is that it be governed by an existing permission, and I think the one I specified suffices for now. 

@regisoc -- in case Ayush has more time, could you elaborate on which group of users you think aren't included/excluded with the 'developer' permission? (It's really a DCC permission, coordinators get it too)

**Comment by christinerogers** [comment]
Look at how buttons are disabled/hidden in other modules depending on a user permission -- Here's an example
- Show/hide Proband button [in the Candidate module ](https://github.com/aces/Loris/blob/32377a205d168683b6542323ca0fd4b99b8aeda7/modules/candidate_parameters/jsx/ProbandInfo.js#L197)

If button style [can be corrected](https://github.com/aces/Loris/issues/9423) at the same time that would be ideal Consider doing  at the same time #9423
